README.md

```diff
@@ -114,103 +108,71 @@ as part of your command:
 Note, if you reference data inside your script, or pass them as arguments to the script, they need to reference the
 aliased path inside the container, here: `/mnt/`, rather than the path `/local/path/`.
 
-### Installation as a Python Package / CLI
+### Installation as a Python Package
 
-You can in your base installation of python or a virtual environment.
 You can use GeNet's CLI to run pre-baked modifications or checks on networks.
-You can also write your own python scripts, importing genet as a package, use IPython shell or Jupyter Notebook to load
-up a network, inspect or change it and save it out to file. Check out the
-[wiki pages](https://github.com/arup-group/genet/wiki/Functionality-and-Usage-Guide) and
-[example jupyter notebooks](https://github.com/arup-group/genet/tree/master/notebooks)
-for usage examples.
-
-**Note:** if you plan only to _use_ GeNet rather than make code changes to it, you can avoid having to perform any
-local installation by using [GeNet's Docker image](#using-docker). If you are going to make code changes or use GeNet's
-CLI locally, follow the steps below:
-
-#### Native dependencies
-GeNet uses some Python libraries that rely on underlying native libraries for things like geospatial calculations and
-linear programming solvers. Before you install GeNet's Python dependencies, you must first install these native
-libraries.
+You can also write your own python scripts, importing genet as a package, use IPython shell or Jupyter Notebook to load up a network, inspect or change it and save it out to file.
+Check out the [wiki pages](https://github.com/arup-group/genet/wiki/Functionality-and-Usage-Guide) and [example jupyter notebooks](https://github.com/arup-group/genet/tree/master/notebooks) for usage examples.
+
+**Note:** if you plan only to _use_ GeNet rather than make code changes to it, you can ignore the rest of this section and instead use [GeNet's Docker image](#using-docker).
+If you are going to make code changes or use GeNet's CLI locally, follow the steps below to [install a development environment](#installing-a-development-environment).
+
+### Installing a development environment
+
+To create a development environment for genet, with all libraries required for development and quality assurance installed, it is easiest to install genet using the [mamba](https://mamba.readthedocs.io/en/latest/index.html) package manager, as follows:
+
+1. Install mamba with the [Mambaforge](https://github.com/conda-forge/miniforge#mambaforge) executable for your operating system.
+2. Open the command line (or the "miniforge prompt" in Windows).
+3. Download (a.k.a., clone) the genet repository: `git clone git@github.com:arup-group/genet.git`
+4. Change into the `genet` directory: `cd genet`
+5. Create the genet mamba environment: `mamba create -n genet -c conda-forge -c city-modelling-lab --file requirements/base.txt --file requirements/dev.txt`
+6. Activate the genet mamba environment: `mamba activate genet`
```

Comment:
I've gone through the install steps, all worked fine except here i needed to use `micromamba activate ..` command instead as `mamba activate ..` did not find the environment (I used `mamba create ..` to create it and it worked fine though..)

Reply:
sounds like you've initialised `micromamba` in your shell on your device (`micromamba init [name-of-shell-of-choice]`) but not `mamba` (similarly, `mamba init [name-of-shell-of-choice]`)